vllm/utils.py

```diff
@@ -163,6 +163,16 @@ def random_uuid() -> str:
     return str(uuid.uuid4().hex)
 
 
+def get_vllm_instance_id():
```

Comment:
This should be cached otherwise repeated call will generate different IDs.

Reply:
fixed in [4636490](https://github.com/vllm-project/vllm/pull/4079/commits/463649038ae964faab2aee4da10be4a327f868d2)